pyanaconda/installation.py

```diff
@@ -68,6 +70,10 @@ def _writeKS(ksdata):
 
     # Make it so only root can read - could have passwords
     with util.open_with_perm(path, "w", 0o600) as f:
+        f.write("# Generated by Anaconda {} with blivet {}\n".format(
```

Comment:
I guess it should be `Anaconda with Blivet` or `anaconda with blivet`.

Reply:
Ah, right. I was not even sure if Blivet has a name... but their docs are unambiguous on that :-)